subsys/mgmt/osdp/src/osdp_cp.c

```diff
@@ -101,37 +101,10 @@ static int cp_build_command(struct osdp_pd *pd, uint8_t *buf, int max_len)
 
 	switch (pd->cmd_id) {
 	case CMD_POLL:
-		if (max_len < CMD_POLL_LEN) {
```

Comment:
Can you add an explicit `__fallthrough`  on this empty cases ? Other than that looks great to me

Reply:
Added.